.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,15 @@
+name: Pull Request
+
+on: [push, pull_request]
```

Comment:
I'd only do PR for now. If you do `push` as well, it'd run twice I think for every push on a PR. Eventually we'll have a workflow for `push` as well, but only to `master` (i.e. PR merges or hot fixes).